# HG changeset patch
# User rmunn
# Date 1505987520 -25200
#      Thu Sep 21 16:52:00 2017 +0700
# Branch 7000069
# Node ID 7dd39bf8fffd26309c796213ac87ccefee4fb902
# Parent  65022e6348169e7e4019b1c802456f6e35a0d6dc
[FLEx Bridge: 2.1] sync Deleted B

diff -r 65022e634816 -r 7dd39bf8fffd Linguistics/Lexicon/Lexicon_02.lexdb
--- a/Linguistics/Lexicon/Lexicon_02.lexdb	Wed Sep 20 15:40:08 2017 +0700
+++ b/Linguistics/Lexicon/Lexicon_02.lexdb	Thu Sep 21 16:52:00 2017 +0700
@@ -1,43 +1,2 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Lexicon>
-	<LexEntry
-		guid="195dd361-8d89-4b3e-a860-fca3cba8216a">
-		<DateCreated
-			val="2017-9-19 9:48:32.63" />
-		<DateModified
-			val="2017-9-19 9:48:32.63" />
-		<DoNotUseForParsing
-			val="False" />
-		<HomographNumber
-			val="0" />
-		<LexemeForm>
-			<MoStemAllomorph
-				guid="8af64994-2594-40a3-9b84-d5679b2e31ae">
-				<Form>
-					<AUni
-						ws="fr">B</AUni>
-				</Form>
-				<IsAbstract
-					val="False" />
-				<MorphType>
-					<objsur
-						guid="d7f713e8-e8cf-11d3-9764-00c04f186933"
-						t="r" />
-				</MorphType>
-			</MoStemAllomorph>
-		</LexemeForm>
-		<Senses>
-			<ownseq
-				class="LexSense"
-				guid="70dff4b7-2469-47a0-9537-6c286d1c664d">
-				<Definition>
-					<AStr
-						ws="en">
-						<Run
-							ws="en">B</Run>
-					</AStr>
-				</Definition>
-			</ownseq>
-		</Senses>
-	</LexEntry>
-</Lexicon>
\ No newline at end of file
+<Lexicon />
\ No newline at end of file
